bluesky/plans.py

```diff
@@ -221,6 +221,431 @@ def single_gen(msg):
     yield msg
 
 
+def create(name='primary'):
+    """
+    Bundle future readings into a new Event document.
+
+    Parameters
+    ----------
+    name : string, optional
+        name given to event stream, used to convenient identification
+        default is 'primary'
+
+    Yields
+    ------
+    msg : Msg
+        Msg('create', name=name)
+    """
+    yield Msg('create', name=name)
+
+
+def save():
+    """
+    Close a bundle of readings and emit a completed Event document.
+
+    Yields
+    -------
+    msg : Msg
+        Msg('save')
+    """
+    yield Msg('save')
+
+
+def read(obj):
+    """
+    Take a reading and add it to the current bundle of readings.
+
+    Parameters
+    ----------
+    obj : Device or Signal
+    
+    Yields
+    ------
+    msg : Msg
+        Msg('read', obj)
+    """
+    yield Msg('read', obj)
+
+
+def monitor(obj, *args, name=None, **kwargs):
+    """
+    Asynchornously monitor for new values and emit Event documents.
+
+    Parameters
+    ----------
+    obj : Signal
+    name : string, optional
+        name of event stream; default is None
+    args :
+        passed through to ``obj.subscribe()``
+    kwargs :
+        passed through to ``obj.subscribe()``
+    
+    Yields
+    ------
+    msg : Msg
+        Msg('monitor', obj, *args, **kwargs)
+    """
+    yield Msg('monitor', obj, *args, **kwargs)
+
+
+def unmonitor(obj):
+    """
+    Stop mointoring.
+
+    Parameters
+    ----------
+    obj : Signal
+
+    Yields
+    ------
+    msg : Msg
+        Msg('unmonitor', obj)
+    """
+    yield Msg('unmonitor', obj)
+
+
+def null():
+    """
+    Yield a no-op Message. (Primarily for debugging and testing.)
+
+    Yields
+    ------
+    msg : Msg
+        Msg('null')
+    """
+    yield Msg('null', obj)
+
+
+def abs_set(obj, *args, group=None, wait=False, **kwargs):
+    """
+    Set a value. Optionally, wait for it to complete before continuing.
+
+    Parameters
+    ----------
+    obj : Device
+    group : string (or any hashable object), optional
+        identifier used by 'wait'
+    wait : boolean, optional
+        If True, wait for completion before processing any more messages.
+        False by default.
+    args :
+        passed to obj.set()
+    kwargs :
+        passed to obj.set()
+
+    Yields
+    ------
+    msg : Msg
+    """
+    yield Msg('set', obj, *args, group=group, **kwargs)
```

Comment:
That's a question for a separate PR, because currently the RunEngine auto-generates. (`None` is a valid group.)